README.md

```diff
@@ -232,6 +232,20 @@ easy to fork and contribute any changes back upstream.
          echo 'export PYENV_ROOT="$HOME/.pyenv"' >> ~/.profile
          echo 'export PATH="$PYENV_ROOT/bin:$PATH"' >> ~/.profile
          echo 'eval "$(pyenv init --path)"' >> ~/.profile
+         ~~~
+            <!--This is an alternative option and needn't be replicated to `pyenv init`-->
+         Alternatively, for an automated installation, you can run the following:
```

Comment:
This alternative is intended for Debian/Ubuntu case only, it won't work in other cases AFAIK. Is there a reason why is was moved?

Reply:
I moved it because this won't work on Ubuntu, since appending the first two lines will mean that it doesn't come before ~/.bashrc is sourced. I figured it was for other OS's - but if that's not the case, maybe it should just be deleted?

Reply:
It was specifically designed for Ubuntu since it's not possible to reliably insert stuff into the middle of other unknown code -- and should work there.

Reply:
but this will insert these lines at the end of the file? so they won't be inserted before ~/.bashrc is sourced which is required?

Reply:
This solution gets around that requirement. If `~/.profile` is sourced as `bash`, the chunk in `~/.bashrc` will run -- still before `eval "$(pyenv init -)"`. If it's executed as `sh`, `~/.bashrc` won't be sourced and the chunk in `~/.profile` will run instead.